src/Stratis.Bitcoin/P2P/Peer/NetworkPeerConnectionParameters.cs

```diff
@@ -98,7 +98,7 @@ public VersionPayload CreateVersion(IPEndPoint peerAddress, Network network, Dat
                 Version = this.Version,
                 Timestamp = timeStamp,
                 AddressReceiver = peerAddress,
-                AddressFrom = this.AddressFrom ?? new IPEndPoint(IPAddress.Parse("0.0.0.0").MapToIPv6Ex(), network.DefaultPort),
+                AddressFrom = externalAddressEndPoint ?? new IPEndPoint(IPAddress.Parse("0.0.0.0").MapToIPv6Ex(), network.DefaultPort),
```

Comment:
We said that we will guarantee that it may not be null

Reply:
I didn't understand this and what correction I needed to make.

Reply:
We wanted to guarantee that it is never null, therefore we don't need `??` And the right side